fix(docs): switched from v4 to evergreen badge link

rebeccaalpert opened the pull request:
Fixes https://github.com/patternfly/patternfly-org/issues/4124.

patternfly-build commented:
Preview: https://patternfly-org-pr-4186-site.surge.sh

mcoker reviewed (approved):
LGTM. A little scope-creep but there is another link on this page that isn't absolute and doesn't take you to the correct page. Just needs a leading `/`

https://github.com/patternfly/patternfly-org/blob/65975ec5c0857f722a6a422e4cb27c2e8c901520/packages/documentation-site/patternfly-docs/content/design-guidelines/components/label/label.md?plain=1#L159

I wonder if we should try to dig up more of those?

rebeccaalpert commented:
> LGTM. A little scope-creep but there is another link on this page that isn't absolute and doesn't take you to the correct page. Just needs a leading `/`
> 
> https://github.com/patternfly/patternfly-org/blob/65975ec5c0857f722a6a422e4cb27c2e8c901520/packages/documentation-site/patternfly-docs/content/design-guidelines/components/label/label.md?plain=1#L159
> 
> I wonder if we should try to dig up more of those?

Sure! Added a leading backslash and fixed the typo there. I haven't looked for more links without slashes, but I am seeing more v4 links that just this. I did a search for `v4` in `./packages/documentation-site/patternfly-docs/content/design-guidelines` and saw some other hits.

mcoker commented on packages/documentation-site/patternfly-docs/content/design-guidelines/components/label/label.md:
```diff
@@ -156,4 +156,4 @@ The new label is created based on the attributes selected by the user and it can
 ## Accessibility
-For information regarding accessibility, visit the [labels accessbility](components/jump-links/accessibility) tab.
\ No newline at end of file
+For information regarding accessibility, visit the [labels accessibility](/components/jump-links/accessibility) tab.
```
```suggestion
For information regarding accessibility, visit the [labels accessibility](/components/label/accessibility) tab.
```

rebeccaalpert replied:
No worries! All set.

mcoker reviewed (changes requested):
So sorry, I didn't even read the a11y link you fixed. Looks like it's also pointing to jump-links a11y tab, and should point to the label tab instead

rebeccaalpert commented:
No worries! Should be all set: https://patternfly-org-pr-4186-site.surge.sh/components/label/design-guidelines

mcoker commented on packages/documentation-site/patternfly-docs/content/design-guidelines/components/label/label.md:
```diff
@@ -156,4 +156,4 @@ The new label is created based on the attributes selected by the user and it can
 ## Accessibility
-For information regarding accessibility, visit the [labels accessbility](components/jump-links/accessibility) tab.
\ No newline at end of file
+For information regarding accessibility, visit the [labels accessibility](components/label/accessibility) tab.
```
Just need to add the leading slash back and I think it's set!

```suggestion
For information regarding accessibility, visit the [labels accessibility](/components/label/accessibility) tab.
```

mcoker reviewed (approved):
FABULOUS! 🤩
